admin/import/plugins/class-import-aioseo-v4.php

```diff
@@ -144,7 +144,7 @@ protected function get_unique_custom_fields_or_taxonomies( $meta_values, $aioseo
 		foreach ( $meta_values as $meta_value ) {
 			// Find all custom field replace vars, store them in `$matches`.
 			\preg_match_all(
-				"/#$aioseo_prefix-(\w+)/",
+				"/#$aioseo_prefix-([\w-]+)/",
```

Comment:
Just a small improvement on the comment of https://github.com/Yoast/wordpress-seo/pull/17446/files#diff-950872bf47d29293fd9ae2088a45223c794e458b8bc28352709361578168abbdR154 since we're changing that regex.

Will push myself